apps/financial-aid/backend/src/app/modules/file/file.controller.ts

```diff
@@ -34,6 +36,7 @@ export class FileController {
   @Get('url/:id')
   @UseGuards(RolesGuard)
   @RolesRules(RolesRule.VEITA)
+  @Scopes(MunicipalitiesFinancialAidScope.read)
```

Comment:
This is still open for attacks.
The scope is set by the client, so it doesn't really guard this. You can get a valid access token (and its scopes) from Veita, even though you don't have access and use that to call this endpoint.
This needs a StaffGuard.